client/src/SearchGithub.jsx

```diff
@@ -105,6 +80,16 @@ const SearchGithub = () => {
   }, []);
 
   useEffect(() => {
+    if (!isSearching) {
```

Comment:
It seems like what you are doing is looking up the statistics of these specific repos, is there a way to actually grab the top 10 most forked github?